syz-cluster/reporter-server/service.yaml

```diff
@@ -4,10 +4,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: reporter-service
+  name: reporter-server-service
 spec:
   selector:
-    app: controller
+    app: reporter-server
```

Comment:
This line looks too different.
Could you please confirm it is correct?

Reply:
Yes, that was a bug that was fixed in this PR.